Neos.ContentGraph.DoctrineDbalAdapter/src/DoctrineDbalContentGraphSchemaBuilder.php

```diff
@@ -107,6 +109,30 @@ private function createReferenceRelationTable(): Table
             ->setPrimaryKey(['name', 'position', 'nodeanchorpoint']);
     }
 
+    private function createWorkspaceTable(): Table
+    {
+        $workspaceTable = self::createTable($this->tableNames->workspace(), [
+            DbalSchemaFactory::columnForWorkspaceName('name')->setNotnull(true),
+            DbalSchemaFactory::columnForWorkspaceName('baseWorkspaceName')->setNotnull(false),
+            DbalSchemaFactory::columnForContentStreamId('currentContentStreamId')->setNotNull(true),
+            (new Column('status', self::type(Types::BINARY)))->setLength(20)->setNotnull(false),
+        ]);
+
+        return $workspaceTable->setPrimaryKey(['name']);
+    }
+
+    private function createContentStreamTable(): Table
+    {
+        return self::createTable($this->tableNames->contentStream(), [
+            DbalSchemaFactory::columnForContentStreamId('id')->setNotnull(true),
```

Comment:
primary key?

Reply:
Good idea, though in a followup id say, this change is big enough already and this was not part of the `ContentStreamProjection`.

Reply:
-> https://github.com/neos/neos-development-collection/pull/5296